scripts/menu.sh

```diff
@@ -0,0 +1,32 @@
+CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+MENU_COMMANDS_FILE="$CURRENT_DIR/menu.sh"
+
+source "$CURRENT_DIR/state.sh"
+
+tmux set-environment -g MENU_COMMANDS_FILE "$MENU_COMMANDS_FILE"
+
+set_theme_option() {
+  local option=$1
+  local value=$2
+
```

Comment:
i would add a line here to check for valid inputs, just in case the function is used later on for something else

Reply:
if more options will be added to the menu later, it will make sense.